content/Teams/observability-platform/sre.md

```diff
@@ -0,0 +1,119 @@
+# Observability Platform Team SRE Processes
+
+This document explains a few processes related to operating our production software together with AppSRE team. All operations can be summarised as the Site Reliability Engineering part of our work.
+
+## Goals
+
+The main goals of our SRE work within this team is to deploy and operate software that meets functional (API) and performance (SLO) requirements of our customers.
+
+Currently, we offer internal service called [RHOBS](../../Projects/Observability/RHOBS) that is used across the company.
+
+## Releasing Changes / Patches
+
+In order to maintain our software on production reliably we need to be able to test, release and deploy our changes rapidly.
+
+We can divide things we change into a few categories. Let's elaborate all processes per each category:
+
+### Software Libraries
+
+Library is a project that is not deployed directly, but rather it is a dependency for [micro-service application](#software-for-micro-services) we deploy. For example https://github.com/prometheus/client_golang or https://github.com/grpc-ecosystem/go-grpc-middleware
+
+*Testing*:
+* Linters, unit and integration tests on every PR.
+
+*Releasing*
+* GitHub release using git tag (RC first, then major/minor or patch release).
+
+*Deploying*
+* Bumping dependency in Service
+* Following on Service [release and deploy steps](#software-for-micro-services).
+
+### Software for (Micro) Services
+
+Software for (micro) services usually lives in many separate open source repositories in GitHub e.g https://github.com/thanos-io/thanos, https://github.com/observatorium/api.
+
+*Testing*:
+* Linters, unit and integration tests on every PR.
+
+*Releasing*
+* GitHub release using git tag (RC first, then major/minor or patch release).
+* Building docker images in quay.io and dockerhub (backup) using CI for each tag or main branch PR.
+
+*Deploying*
+* Bumping docker tag version in configuration.
+* Following [configuration release and deployment steps](#configuration).
+
+### Configuration
+
+All or configuration is rooted in https://github.com/rhobs/configuration configuration templates written in jsonnet. It can be then overridden by defined parameters in app-interface saas files  .
```

Comment:
```suggestion
All or configuration is rooted in https://github.com/rhobs/configuration configuration templates written in jsonnet. It can be then overridden by defined parameters in App-interface Saas files.
```